modules/ROOT/pages/configure-headers4-task.adoc

```diff
@@ -1,37 +1,73 @@
-= To Configure Headers
+= Configuring the Response Headers and Status Code
```

Comment:
In this case, gerund is correct - there are multiple tasks (imperative) in this file.

Reply:
@hannanelson, thanks for your feedback!
I understand that we should use gerund in H1 and infinitive in H2+.
Actually, the APIkit beta site (https://dev-docs-internal.kqa.msap.io/beta-apikit-new-content-structure/apikit/4.x/overview) has been structured like that.
Please, can you confirm if gerund is OK for H1 and infinitive for H2+?
Thanks!

Reply:
Yes, if there are multiple tasks in a file.

Reply:
In this file, there is more than one task, so I understand that it is OK to use the structure gerund for H1 and infinitive for H2. 
Thanks for your support!

Reply:
Yes.